### DISCUSSION TITLE
Delete connection inside a project through API using python

### DISCUSSION THREAD

**Discussion by gerso-magro** [idea-statica/ideastatica-public#646] [Help Q&A] [answered]
Hello, I am looking for a way to delete a connection inside of a file that has 3 connections. I've been looking for a method that helps me to do so, but i couldn't find one. 
I am using idea statica v 24

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @gerso-magro, it is not currently possible. I have added it as a request here. https://github.com/idea-statica/ideastatica-public/issues/647
